### PR TITLE
Added testing module to help with test creation

### PR DESCRIPTION
**Pull request by Mikkel-Boje**

### CODE REVIEW

File: server/src/managers/message_persister.rs [hunk 1]
```diff
@@ -270,7 +270,8 @@ mod message_persister_tests {
         message_times: Vec<u64>,
     ) -> (bool, bool, bool, Vec<String>, Vec<String>) {
         let _ = dotenv::dotenv();
-        let db = PostgresDatabase::connect("DATABASE_URL".to_string()).await;
+        let db = PostgresDatabase::connect("DATABASE_URL".to_string()).await; // Should this not be
```
Comment (alexsteffensen) [inline server/src/managers/message_persister.rs:273]:
```suggestion
        let db = PostgresDatabase::connect("DATABASE_URL_TEST".to_string()).await; // Should this not be
```

**Review by alexsteffensen — DISMISSED** [review]
Det ser fandme lækkert ud

File: server/src/test_utils/user_mocks.rs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+use common::web_api::{AccountAttributes, DeviceCapabilities};
+use libsignal_core::{Pni, ProtocolAddress};
+use libsignal_protocol::{IdentityKey, KeyPair, PublicKey};
+use rand::{
+    rngs::{OsRng, StdRng},
+    Rng, SeedableRng,
+};
+use uuid::Uuid;
+
+pub async fn connect() -> PostgresDatabase {
```
Comment (MJHC) [inline server/src/test_utils/user_mocks.rs:14]: please rename to database_connect

Comment (MJHC) [reply server/src/test_utils/user_mocks.rs:14]: and move to database helpers

**Review by MJHC — CHANGES_REQUESTED** [review]
please rename test_utils subfiles to its corresponding names instead of having post fixes as we already know they are for tests when they are in this dir

**Review by alexsteffensen — APPROVED** [review]
LGTM

**Review by MJHC — APPROVED** [review]
okay

**Pull request merged**